.github/workflows/haskell.yml

```diff
@@ -50,52 +52,56 @@ jobs:
             cabal_build_options: "--allow-newer=hsc2hs"
             disable_sdist_build: "y"
             runner: ubuntu-latest
-          - name: 8.10.4+stack+lts-17.8
+          - name: 8.10.4+stack+nightly-2021-06-10
             build: stack
-            resolver: lts-17.8
+            resolver: nightly-2021-06-10
+            stack_yaml: stack.yaml
             sdist_options: "--ignore-check"
             stack_build_options: "--flag streamly-benchmarks:-opt"
             runner: ubuntu-latest
-          - name: 8.10.2+macOS
-            ghc_version: 8.10.2
+          - name: 8.10.4+macOS
+            ghc_version: 8.10.4
             build: cabal-v2
             disable_sdist_build: "y"
             runner: macos-latest
-          - name: 8.8.3+inspection+fusion-plugin+Werror
-            ghc_version: 8.8.3
+          - name: 8.8.4+inspection+fusion-plugin+Werror
+            ghc_version: 8.8.4
             build: cabal-v2
             cabal_project: cabal.project.ci
             cabal_build_options: "--flag fusion-plugin --flag inspection"
             runner: ubuntu-latest
-          - name: 8.8.4+stack+lts-16.31
-            build: stack
-            resolver: lts-16.31
-            sdist_options: "--ignore-check"
-            stack_build_options: "--flag streamly-benchmarks:-opt"
+          - name: 8.8.4
+            ghc_version: 8.8.4
+            build: cabal
+            cabal_project: cabal.project
```

Comment:
Don't you need to disable sdist build if you want to include `cabal.project`?

Reply:
CABAL_PROJECT if specified is used irrespective of sdist build or not.